tests/batching_test.py

```diff
@@ -989,6 +989,9 @@ def testCommAssocCollective(self, collective, bulk_op, vmap_names, collective_na
       f = vmap(f, axis_name=axis_name)
     self.assertAllClose(f(x), x - bulk_op(x, axis=tuple(range(len(vmap_names)))))
 
+    if collective is lax.psum:
+      jtu.check_grads(f, (x,), 2, eps=1)
```

Comment:
Just double-checking: would this test have caught the issue that the user reported?

Reply:
Yes, it would.